sympy/utilities/tests/test_lambdify.py

```diff
@@ -1636,11 +1636,17 @@ def test_deprecated_set():
     with warns_deprecated_sympy():
         lambdify({x, y}, x + y)
 
+
 def test_23536_lambdify_cse_dummy():
 
     f = Function('x')(y)
     g = Function('w')(y)
     expr = z + (f**4 + g**5)*(f**3 + (g*f)**3)
     expr = expr.expand()
     eval_expr = lambdify(((f, g), z), expr, cse=True)
-    eval_expr((1.0, 2.0), 3.0)
+    ans = eval_expr((1.0, 2.0), 3.0)  # shouldn't raise NameError
+
+    if not numpy:
+        skip("numpy not installed.")
+    else:
+        numpy.testing.assert_allclose(ans, 300.0)
```

Comment:
There is a sympy version of this function in `sympy.testing.matrices`.

Reply:
That only works for matrices. `ans` is not a matrix.